### ISSUE TITLE
Support file copy op?

### ISSUE THREAD

**Issue by SerhoLiu** [tencentyun/cos-python-sdk-v4#24]
看官网文档是支持文件 Copy 操作的 https://www.qcloud.com/document/product/436/7419

不过 SDK 没有实现?

**Comment by liuchang0812** [comment]
Yep, it's supported by Python SDK. The documentation of SDK need to be updated. thanks
